src/Components/Publishing/Sections/ImageCollection.tsx

```diff
@@ -20,7 +20,7 @@ interface ImageCollectionProps {
   }
 }
 
-class ImageCollectionComponent extends React.Component<ImageCollectionProps, null> {
+class ImageCollectionComponent extends React.PureComponent<ImageCollectionProps, null> {
```

Comment:
@kanaabe, @eessex - [`PureComponent`](https://reactjs.org/docs/react-api.html#reactpurecomponent) is how I fixed the repetitive image fade-in issue. Since this content will only be set on initial render no need to allow re-renders (unless data changes). 

If you're ever running into perf issues this really helps, but I would consider it a premature optimization to be avoided until necessary or your component data-structure is entirely flat without nesting (as `PureComponent` only does shallow comparisons).

Reply:
👍 Very nice - I was wondering about the repetitive image fade, thanks!